script.grab.fanart/resources/lib/utils.py

```diff
@@ -0,0 +1,25 @@
+from kodi_six import xbmc, xbmcaddon, xbmcgui
+
+__addon_id__= 'script.grab.fanart'
+__Addon = xbmcaddon.Addon(__addon_id__)
+
+def data_dir():
+    return __Addon.getAddonInfo('profile')
+
+def addon_dir():
+    return __Addon.getAddonInfo('path')
+
+def log(message,loglevel=xbmc.LOGDEBUG):
+    xbmc.log(__addon_id__ + "-" + __Addon.getAddonInfo('version') +  ": " + message,level=loglevel)
+
+def showNotification(message):
+    xbmcgui.Dialog().notification(getString(30010),message,time=4000,icon=xbmc.translatePath(__Addon.getAddonInfo('path') + "/resources/media/icon.png"))
+
+def getSetting(name):
```

Comment:
You now have a few more methods to get settings as books, ints, etc to avoid uneeded conversion

Reply:
Good call, I've used them in other addons but this one only has 2 settings. One is an number though so I'll adjust.